mne/tests/test_epochs.py

```diff
@@ -1220,21 +1234,27 @@ def test_evoked_io_from_epochs(tmpdir):
     assert_allclose(evoked.data, evoked2.data, rtol=1e-4, atol=1e-20)
     assert_allclose(evoked.times, evoked2.times, rtol=1e-4,
                     atol=1 / evoked.info['sfreq'])
+    # assert evoked2.baseline == baseline XXX find I/O roundtrip
```

Comment:
Cruft? Should it match now?

Reply:
Good catch, I had temporarily disabled that for testing. Took care of it in fbf112f!